#! /bin/sh
patch -p1 -f $* < $0
exit $?

Add a Hurd machine target.  We configure the Linux directory and then
override what we do not want.

diff -upr newlib-1.16.0/newlib/libc/sys/configure.in newlib/newlib/libc/sys/configure.in
--- newlib-1.16.0/newlib/libc/sys/configure.in  2007-05-24 19:33:37.000000000 +0200
+++ newlib/newlib/libc/sys/configure.in 2008-01-17 16:13:59.000000000 +0100
@@ -29,6 +29,7 @@ if test -n "${sys_dir}"; then
 	h8300hms) AC_CONFIG_SUBDIRS(h8300hms) ;;
 	h8500hms) AC_CONFIG_SUBDIRS(h8500hms) ;;
 	linux) AC_CONFIG_SUBDIRS(linux) ;;
+	hurd) AC_CONFIG_SUBDIRS(hurd) ;;
 	m88kbug) AC_CONFIG_SUBDIRS(m88kbug) ;;
 	mmixware) AC_CONFIG_SUBDIRS(mmixware) ;;
 	netware) AC_CONFIG_SUBDIRS(netware) ;;
